src/ui/zcl_abapgit_gui_router.clas.abap

```diff
@@ -22,6 +22,12 @@ CLASS zcl_abapgit_gui_router DEFINITION
       RETURNING VALUE(ri_page) TYPE REF TO zif_abapgit_gui_page
       RAISING   zcx_abapgit_exception.
 
+    METHODS get_page_patch
+      IMPORTING iv_getdata     TYPE clike
+                iv_prev_page   TYPE clike
+      RETURNING VALUE(ri_page) TYPE REF TO zif_abapgit_gui_page
+      RAISING   zcx_abapgit_exception.
+
```

Comment:
can this be added in the pages calling the page instead? how data is encoded is handled by the page, so the decoding is also page dependent. Router is central point of merge conflicts

Reply:
method is moved